File: gacsach/toc.lua
local url = ...
local list = {}
local doc = http:get(url):html()
if doc ~= nil then
    local name = doc:select("h1.page-title"):text() .. " - "
    local el = doc:select("div.book-nav ul.menu li a")
    for i = 0, el:size() - 1 do
        local e = el:get(i)
        local chap = {}
        chap["name"] = text:remove(e:text(), { name })
        chap["url"] = e:attr("href")
        chap["host"] = "https://gacsach.com"
        table.insert(list, chap)
    end
    return response:success(list)
end
return nil

File: hentaivn/toc.js
function execute(url) {
    var doc = Http.get(url).html();

     var el = doc.select(".listing a")
    const data = [];
    for (var i = el.size() - 1; i >= 0 ; i--) {
        var e = el.get(i);
        data.push({
            name: e.text(),
            url: e.attr("href"),
            host: "https://hentaivn.net"
        })
    }

    return Response.success(data);
}